src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -872,12 +875,14 @@ def onDrawClicked(self):
                     set()
                 )
                 self.slice_changed = True
+                self.has_drawing = True
                 self.dicom_view.view.setScene(self.drawingROI)
                 self.enable_cursor_diameter_change_box()
             else:
                 QMessageBox.about(self.draw_roi_window_instance,
                                   "Not Enough Data",
                                   "Not all values are specified or correct.")
+        logging.debug("onFillClicked finished")
```

Comment:
In this case it makes sense because of the process in between. I see that you probably added the other debug for consistency. nice job tho